IREC/IREC_PDD.tex

```diff
@@ -0,0 +1,288 @@
+
+\subsection{Deliverables}
+\label{subsec:Deliverables}
+  % When all is said and done, what will you have to show for it?
+  % Examples: Hardware, software, poster, ImagineRIT demo, presentations, technical papers...
+The first deliverable will be the preliminary design and bill of materials with a corresponding estimate of size, weight, and
+cost. If it is found that additional funds need to be raised, then the preliminary design can be used in fundraising requests
+so that businesses and investors know that sufficient research and development has already been put into the project. After the
+preliminary designs, the next deliverable will be the final external dimensions and interfaces so that LI can proceed with their
+construction of the rocket. Finally, the actual payload will need to be delivered by the competition date although it will be provided
+much earlier than that due to the necessity of integration and testing.
+
+\section{Externalities}
+  % Things not directly related to the work or outcomes, but related to the project as a whole.
+\subsection{Prerequisite Skills}
+  % Which skills do team members need to have before work can start (not including skills that will be learned ``on the job'')?
+To participate in this endeavor, there are no prerequisite skills. However, for the project to succeed, there will need to be
+individuals who are proficient in computer-aided design and analysis for the mechanical structure of the payload. There will
+also need to be individuals proficient in electronic design, specifically with respect to printed circuit boards. Additionally,
+individuals proficient in programming will be required for embedded systems design and ground-station interfaces. Finally,
+there will need to be some individuals who are willing to spend time researching IREC rules, previous related
+payload missions, and acting as a resource to those designing the hardware and software.
+
+\subsection{Funding Requirements}
+  % Estimate costs that would be needed to meet objectives.
+Without knowing the target payload, it is difficult to give an accurate funding requirement. However, for ease of financial
+planning and design constraints, the target value is 500 dollars. If additional funding is needed after a sufficient effort
+has been provided, then the SPEX admin team will be contacted and steps will be taken to close the gap and secure the necessary
+funding.
+
+\subsection{Faculty Support}
+  % Identify faculty that will be involved (or would need to be involved) to meet objectives.
+  % Note that if a professor is the Principal Investigator (P.I.) for a project, there still needs to be a student as the SPEX Project Champion.
+Faculty support will be consistently saught after during the design and manufacturing of the payload. Although, as discussed before,
+no faculty have yet been willing to dedicate a large block of time to assist this project. There have been multiple offers to provide
```

Comment:
Is it appropriate to write the current state?

Reply:
No